sklearn/manifold/_t_sne.py

```diff
@@ -491,7 +493,23 @@ def trustworthiness(X, X_embedded, *, n_neighbors=5, metric="euclidean"):
     -------
     trustworthiness : float
         Trustworthiness of the low-dimensional embedding.
+
+    References
+    ----------
+    .. [1] Jarkko Venna and Samuel Kaski. 2001. Neighborhood
```

Comment:
With references down here, can we remove the references above?

https://github.com/scikit-learn/scikit-learn/blob/b4da3b406379b241bf5e81d0f60bbcddd424625b/sklearn/manifold/_t_sne.py#L461-L465